server/app/services/applications/ApplicationAlreadyInStatus.java

```diff
@@ -0,0 +1,9 @@
+package services.applications;
+
+public class ApplicationAlreadyInStatus extends RuntimeException {
+  ApplicationAlreadyInStatus(Long applicationId, String status) {
+    super(
+        String.format(
+            "Application for the id %d, is in already in the status %s.", applicationId, status));
```

Comment:
**nitpick:** Typo. There's one too many "in"s.

Reply:
good catch!